examples/00-mapdl-examples/bracket_static.py

```diff
@@ -0,0 +1,743 @@
+# using :class:`mapdl.solution() <ansys.mapdl.core.solution.Solution>`
+# or the APDL :meth:`mapdl.slashsolu() <ansys.mapdl.core.Mapdl.slashsolu>`
+# command. This was done a few steps earlier.
+#
+# The model is ready to be solved using the
+# :meth:`mapdl.solve() <ansys.mapdl.core.Mapdl.solve>` command.
+#
+
+# Solve the model
+output = mapdl.solve()
+print(output)
+
+###############################################################################
+# Mechanical APDL stores the results of this single-load-step problem in the
+# database and in the results file, :file:`Jobname.RST` (or :file:`Jobname.RTH`
+# for thermal, :file:`Jobname.RMG` for magnetic). The database can contain only
+# one set of results at any given time, so in a multiple-load-step or
+# multiple-substep analysis, Mechanical APDL stores only the final solution in
+# the database.
+#
+# Mechanical APDL stores all solutions in the results file.
+#
+# Review the results
+# ==================
+#
+# This step represents the beginning of the postprocessing phase.
+#
+# .. note:: The results you see may vary slightly from what is shown due to variations in the mesh.
+#
+# Enter the postprocessor
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# The Ansys APDL postprocessor is a separate processor called with the
+# :meth:`mapdl.post1() <ansys.mapdl.core.Mapdl.post1>` command.
+#
+
+mapdl.post1()
+
+###############################################################################
+# Plot the deformed shape
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# Here :class:`mapdl.result <ansys.mapdl.core.Mapdl.result>` is used to retrieve
+# the results and for plotting.
+#
+
+# Plot displacement
+result = mapdl.result
+result_set = 0  # Plotting the first results
+disp_fact = 1e10
+result.plot_nodal_displacement(
+    result_set,
+    cpos="xy",
+    displacement_factor=5,
+    show_displacement=True,
+    show_edges=True,
+)
+
+###############################################################################
+# Plot the von mises equivalent stress
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Stress plots are also available using the
+# :meth:`mapdl.plot_principal_nodal_stress() <ansys.mapdl.core.Mapdl.plot_principal_nodal_stress>`
+# command.
+#
+
+result.plot_principal_nodal_stress(
+    0,
+    "SEQV",
+    cpos="xy",
+    background="w",
+    text_color="k",
+    add_text=True,
+    show_edges=True,
+)
+
+
+###############################################################################
+# Obtaining the von misses stresses
+#
+nnum, stress = result.principal_nodal_stress(0)
+# von-Mises stress is the last column in the stress results [-1]
+von_mises = stress[:, -1]
+
+
+###############################################################################
+# List reaction solution
+# ~~~~~~~~~~~~~~~~~~~~~~
+#
+# To list the FY reactions forces use the APDL
+# :meth:`mapdl.prrsol() <ansys.mapdl.core.Mapdl.prrsol>` command which print
+# the constrained node reaction solution.
```

Comment:
```suggestion
# To list the FY reactions forces, use the APDL
# :meth:`mapdl.prrsol() <ansys.mapdl.core.Mapdl.prrsol>` command, which prints
# the constrained node reaction solution.
```
Will users know what FY reactions are? I know I don't Does "FY" need to be spelled out?

Reply:
True.